ppr-ui/src/components/common/CourtOrder.vue

```diff
@@ -138,6 +138,8 @@
               title="Select the date of the order"
               clearable
               :errorMsg="errors.orderDate.message ? errors.orderDate.message : ''"
+              :initialValue="orderDate"
```

Comment:
Tested in the date picker component?

Reply:
yep -- same with the other changes

Reply:
just had that merged/published in the newer version

Reply:
cool